core/src/assets/luigi-config.js

```diff
@@ -263,25 +264,125 @@ function getNodes(context) {
       ]
     }
   ];
+  return Promise.all([
+    getUiEntities('microfrontends', environment),
+    getUiEntities('clustermicrofrontends', undefined, 'environment')
```

Comment:
additionally, we need to fetch also cmf with placement = `namespace` because of the renaming story:
```suggestion
    getUiEntities('clustermicrofrontends', undefined, 'environment'),
    getUiEntities('clustermicrofrontends', undefined, 'namespace')
```